packages/ui-core/src/components/slider/interval-slider.component.tsx

```diff
@@ -103,10 +104,41 @@ export const IntervalSlider: FC<Props> = ({
 
   const stepDimension = dimension / intervals.length;
 
+  const handleClick = (e: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
Refactor it to `useCallback` with proper arguments like `x`, `event` and dependencies array - to avoid re-defining that function for each re-render.